.gitignore

```diff
@@ -264,3 +264,77 @@ __pycache__/
 
 # Milliorn
 .vscode/settings.json
+/Module/uti/xp_tome_005.uti.json
```

Comment:
These files should not be git ignored. Please undo the changes to this file.

Reply:
That's done!